doc/user_guide/electron_holography.rst

```diff
@@ -0,0 +1,26 @@
+Electron Holography

```

Comment:
Maybe this new section of the documentation should probably be added in #1079 instead and this one document the new methods in `BaseSignal`, `Signal2D` and the new wave signal?